packages/calling-stateful-client/src/StatefulCallClient.ts

```diff
@@ -232,11 +232,14 @@ export type StatefulCallClientOptions = {
  * @returns
  */
 export const createStatefulCallClient = (
-  args: StatefulCallClientArgs,
+  args?: StatefulCallClientArgs,
   options?: StatefulCallClientOptions
 ): StatefulCallClient => {
   const callClient = new CallClient(options?.callClientOptions);
-  const context: CallContext = new CallContext(args.userId, options?.maxStateChangeListeners);
+  const context: CallContext = new CallContext(
+    args?.userId ?? 'contoso_user_id_not_set',
```

Comment:
"Contoso" is more of an internal and presentation placeholder word, not so good in code.

It's a little weird that the state/context has userId as mandatory but we fake make it optional by populating it with a placeholder.

Reply:
If we want to make this fully optional we would need to change the CallClientState and CallContext userId to be optional too. This might need some changes in selectors and maybe Calling Sample too.